docs/hooks.md

```diff
@@ -98,7 +98,7 @@ Following is description is in a [MSON](https://github.com/apiaryio/mson) format
 
 - transaction (object)
     - name: `"Hello, world! > Retrieve Message"` (string) Transaction identification name used for referencing
-    - host: `"localhost"` (string) hostname without port
+    - host: `"http://localhost"` (string) hostname without port
```

Comment:
This was also correct in the previous version (see `protocol: ...` couple of lines below).